docs/02-guides/customize-the-generated-sitemap.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: Customize the generated sitemap
+description:
+  In this guide you will learn how to customize the generated sitemap.
+---
+
+<SinceVersion tag="3.4" />
+
+<p>{frontMatter.description}</p>
+
+## Static Pages Customisation
+
+You can customize the sitemap for static pages by using the `getSitemapEntries`
+function in the SEO handle. This function allows you to add custom entries to
+the sitemap. You can use this function to add custom URLs to the sitemap, set
+the last modification date, change frequency, and priority for each URL.
```

Comment:
**nitpick:** The 2nd sentences feels redundant to me

**suggestion:** remove it
```suggestion
function in the SEO handle. You can use this function to add custom URLs to the sitemap, set
the last modification date, change frequency, and priority for each URL.
```